docs/en/aws-firehose/aws-firehose-setup.asciidoc

```diff
@@ -103,8 +116,16 @@ A duration of 60-300s should be suitable for most use cases.
 
 * Elastic recommends setting the *`es_datastream_name`* parameter to help route data to the correct integration data streams.
 If this parameter is not specified, data is sent to the *`logs-generic-default`* data stream by default.
+
+[WARNING]
+====
+The default data stream will change to `logs-awsfirehose-default` in the near future.
```

Comment:
Are we able to be more explicit here about timings? I also had a go at making the instructions a little clearer, what do you think? 

> The default data stream will change to `logs-awsfirehose-default` in January 2024. **To avoid breaking changes, do not leave `es_datastream_name` empty.** If you want to try the new [routing functionality](https://docs.elastic.co/integrations/awsfirehose), you should explicitly set `es_datastream_name` to `logs-awsfirehose-default`.